server/db/seed.js

```diff
@@ -108,7 +108,7 @@ const seed = {
   userSeed
 };
 
-syncAndSeed();
+// syncAndSeed();

```

Comment:
Why did we comment out this line?
I thought the whole point of this file was to actually do the seeding of the db?

Reply:
It was for testing purposes, I went ahead and pushed again, so that seed.js has syncAndSeed() uncommented. Now it the test file I commented the syncAndSeed() out there because the can conflict and get validationerrors. So as it is now if we want to npm run test you might have to go into the seed.js dir and run 'node seed.js' to reseed before testing to get all 8 tests to pass. So yeah I think when we merge to dev we should have syncAndSeed() uncommented in the seed.js file.

Reply:
Maybe its better in package.json to pass an env variable like
`"test:watch": "NO_SEED=1 jest`
or similar for "test:watch"

then in the seed.js file do something like
`if(!NO_SEED) syncAndSeed()`

Does that seem reasonable? That would kill both birds with one stone I think...

Reply:
yeah! That gave me the idea to add node seed.js to the package.json test script, so now it's working both ways because it node seeds when a test command is run. I pushed it up, it now reads:     "test": "node server/db/seed.js & jest --detectOpenHandles"

Reply:
You ROCK!!! Thank you!!!